pkg/operator/starter.go

```diff
@@ -104,6 +112,37 @@ func RunOperator(clientConfig *rest.Config, stopCh <-chan struct{}) error {
 	return fmt.Errorf("stopped")
 }
 
+func StaticPodIsRunningFunc(podlister corev1listers.PodLister, ns string) installer.StaticPodInstallStateFunc {
+	return func(installerPod *corev1.Pod, deploymentID int32, nodeName string) (installer.StaticPodInstallState, error) {
+		switch installerPod.Status.Phase {
+		case corev1.PodSucceeded:
+			// installer succeeded, wait for running static pod
+			pod, err := podlister.Pods(ns).Get("openshift-kube-apiserver-" + nodeName)
```

Comment:
They are likely to all be the same but for name and namespace, right?  We already pass the namespace, we could pass a name prefix.